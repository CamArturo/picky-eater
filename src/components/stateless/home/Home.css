.discover-tagline {
  background-color: #b22012;
  padding: 1.7rem;
  margin: 0;
  color: #fff;
  opacity: .85;
  text-align: center;
  font-family: 'Signika Negative', serif;
  letter-spacing: 1px;
  font-size: 1.4rem; }

.discover-info h3 {
  font-family: Signika Negative, serif;
  font-style: normal;
  font-weight: bold;
  line-height: 38px;
  font-size: 24px;
  letter-spacing: 0.04em;
  margin: 0;
  text-align: center;
  color: #353131; }
.discover-info .infographics {
  justify-content: space-around;
  margin: 20px 0 0 0;
  padding-bottom: 100px; }
.discover-info .icon-container {
  display: flex;
  flex-wrap: wrap;
  align-items: center;
  justify-content: center;
  padding: 1.5rem;
  width: 25%; }
  .discover-info .icon-container img {
    height: 120px; }
  .discover-info .icon-container p {
    margin: 2rem 0 0 0;
    text-align: center;
    font-size: 1.2rem;
    font-family: "Mukta Vaani", sans-serif; }

/* Large Devices, Wide Screens */
/* Medium Devices, Desktops */
/* Small Devices, Tablets */
/* Extra Small Devices, Phones */
/* Custom, iPhone Retina */

/*# sourceMappingURL=Home.css.map */
