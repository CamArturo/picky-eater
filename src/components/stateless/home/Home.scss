.discover-tagline {
  background-color: #b22012;
  padding: 2rem;
  margin: 0;
  color: #fff;
  opacity: .9;
  text-align: center;
  font-family: 'Signika Negative', serif;
  letter-spacing: 1px;
  font-size: 1.5rem;
}

.discover-info {
  h3 {
    font-family: Signika Negative, serif;
    font-style: normal;
    font-weight: bold;
    line-height: 38px;
    font-size: 24px;
    letter-spacing: 0.04em;
    margin: 0;
    text-align: center;

    color: #353131;
    // 0956B2
  }
  .infographics {
    justify-content: space-around;
    margin: 20px 0 0 0;
    padding-bottom: 100px;
  }
  .icon-container {
    display: flex;
    flex-wrap: wrap;
    align-items: center;
    justify-content: center;
    padding: 1.5rem;
    width: 25%;
    img {
      height: 120px;
    }
    p {
      margin: 2rem 0 0 0;
      text-align: center;
      font-size: 1.2rem;
      font-family: 'Mukta Vaani', sans-serif
    }
  }
}

/* Large Devices, Wide Screens */
@media only screen and (max-width: 1200px) {

}

/* Medium Devices, Desktops */
@media only screen and (max-width: 992px) {

}

/* Small Devices, Tablets */
@media only screen and (max-width: 768px) {

}

/* Extra Small Devices, Phones */
@media only screen and (max-width: 480px) {

}

/* Custom, iPhone Retina */
@media only screen and (max-width: 320px) {

}