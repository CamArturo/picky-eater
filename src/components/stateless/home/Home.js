import React from 'react';
import './Home.css';
import location from '../../../assets/location-17-240.png';
import Hero from '../hero/Hero';

const Home = () => {
  return (
    <div>
      <Hero></Hero>
      <section className="discover-info">
        <h3>There is a right choice after all.</h3>
        <section className="infographics d-flex">
          <img src={location} alt="Location" />
          <img src={location} alt="Location" />
          <img src={location} alt="Location" />
        </section>
      </section>
    </div>
  );
};

export default Home;
