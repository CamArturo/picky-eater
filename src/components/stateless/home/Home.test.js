import React from 'react';
import Home from './Home';
import { shallow } from 'enzyme';


it('Should match the snapshot', () => {
  const appComponent = shallow(<Home />);

  expect(appComponent).toMatchSnapshot();
});
