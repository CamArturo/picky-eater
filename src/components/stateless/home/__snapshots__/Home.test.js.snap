// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <withRouter() />,
        <h2
          className="discover-tagline"
>
          When nothing sounds good, let us help you find your perfect meal.
</h2>,
        <section
          className="discover-info"
>
          <section
                    className="infographics d-flex"
          >
                    <div
                              className="icon-container"
                    >
                              <img
                                        alt="Location"
                                        src="hungry.png"
                              />
                              <p>
                                        Hungry but you can't put your finger on what you want?
                              </p>
                    </div>
                    <div
                              className="icon-container"
                    >
                              <img
                                        alt="Location"
                                        src="map.png"
                              />
                              <p>
                                        Sort the restaurants in your area by process of elimination.
                              </p>
                    </div>
                    <div
                              className="icon-container"
                    >
                              <img
                                        alt="Location"
                                        src="food-drink.png"
                              />
                              <p>
                                        Discover what you been craving.
                              </p>
                    </div>
                    <div
                              className="icon-container"
                    >
                              <img
                                        alt="Location"
                                        src="smiley.png"
                              />
                              <p>
                                        Come back next time you're hungry!
                              </p>
                    </div>
          </section>
</section>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "When nothing sounds good, let us help you find your perfect meal.",
          "className": "discover-tagline",
        },
        "ref": null,
        "rendered": "When nothing sounds good, let us help you find your perfect meal.",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <section
            className="infographics d-flex"
>
            <div
                        className="icon-container"
            >
                        <img
                                    alt="Location"
                                    src="hungry.png"
                        />
                        <p>
                                    Hungry but you can't put your finger on what you want?
                        </p>
            </div>
            <div
                        className="icon-container"
            >
                        <img
                                    alt="Location"
                                    src="map.png"
                        />
                        <p>
                                    Sort the restaurants in your area by process of elimination.
                        </p>
            </div>
            <div
                        className="icon-container"
            >
                        <img
                                    alt="Location"
                                    src="food-drink.png"
                        />
                        <p>
                                    Discover what you been craving.
                        </p>
            </div>
            <div
                        className="icon-container"
            >
                        <img
                                    alt="Location"
                                    src="smiley.png"
                        />
                        <p>
                                    Come back next time you're hungry!
                        </p>
            </div>
</section>,
          "className": "discover-info",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="icon-container"
>
                <img
                                alt="Location"
                                src="hungry.png"
                />
                <p>
                                Hungry but you can't put your finger on what you want?
                </p>
</div>,
              <div
                className="icon-container"
>
                <img
                                alt="Location"
                                src="map.png"
                />
                <p>
                                Sort the restaurants in your area by process of elimination.
                </p>
</div>,
              <div
                className="icon-container"
>
                <img
                                alt="Location"
                                src="food-drink.png"
                />
                <p>
                                Discover what you been craving.
                </p>
</div>,
              <div
                className="icon-container"
>
                <img
                                alt="Location"
                                src="smiley.png"
                />
                <p>
                                Come back next time you're hungry!
                </p>
</div>,
            ],
            "className": "infographics d-flex",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="Location"
                    src="hungry.png"
/>,
                  <p>
                    Hungry but you can't put your finger on what you want?
</p>,
                ],
                "className": "icon-container",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Location",
                    "src": "hungry.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Hungry but you can't put your finger on what you want?",
                  },
                  "ref": null,
                  "rendered": "Hungry but you can't put your finger on what you want?",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="Location"
                    src="map.png"
/>,
                  <p>
                    Sort the restaurants in your area by process of elimination.
</p>,
                ],
                "className": "icon-container",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Location",
                    "src": "map.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Sort the restaurants in your area by process of elimination.",
                  },
                  "ref": null,
                  "rendered": "Sort the restaurants in your area by process of elimination.",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="Location"
                    src="food-drink.png"
/>,
                  <p>
                    Discover what you been craving.
</p>,
                ],
                "className": "icon-container",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Location",
                    "src": "food-drink.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Discover what you been craving.",
                  },
                  "ref": null,
                  "rendered": "Discover what you been craving.",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="Location"
                    src="smiley.png"
/>,
                  <p>
                    Come back next time you're hungry!
</p>,
                ],
                "className": "icon-container",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Location",
                    "src": "smiley.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Come back next time you're hungry!",
                  },
                  "ref": null,
                  "rendered": "Come back next time you're hungry!",
                  "type": "p",
                },
              ],
              "type": "div",
            },
          ],
          "type": "section",
        },
        "type": "section",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <withRouter() />,
          <h2
            className="discover-tagline"
>
            When nothing sounds good, let us help you find your perfect meal.
</h2>,
          <section
            className="discover-info"
>
            <section
                        className="infographics d-flex"
            >
                        <div
                                    className="icon-container"
                        >
                                    <img
                                                alt="Location"
                                                src="hungry.png"
                                    />
                                    <p>
                                                Hungry but you can't put your finger on what you want?
                                    </p>
                        </div>
                        <div
                                    className="icon-container"
                        >
                                    <img
                                                alt="Location"
                                                src="map.png"
                                    />
                                    <p>
                                                Sort the restaurants in your area by process of elimination.
                                    </p>
                        </div>
                        <div
                                    className="icon-container"
                        >
                                    <img
                                                alt="Location"
                                                src="food-drink.png"
                                    />
                                    <p>
                                                Discover what you been craving.
                                    </p>
                        </div>
                        <div
                                    className="icon-container"
                        >
                                    <img
                                                alt="Location"
                                                src="smiley.png"
                                    />
                                    <p>
                                                Come back next time you're hungry!
                                    </p>
                        </div>
            </section>
</section>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "When nothing sounds good, let us help you find your perfect meal.",
            "className": "discover-tagline",
          },
          "ref": null,
          "rendered": "When nothing sounds good, let us help you find your perfect meal.",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <section
              className="infographics d-flex"
>
              <div
                            className="icon-container"
              >
                            <img
                                          alt="Location"
                                          src="hungry.png"
                            />
                            <p>
                                          Hungry but you can't put your finger on what you want?
                            </p>
              </div>
              <div
                            className="icon-container"
              >
                            <img
                                          alt="Location"
                                          src="map.png"
                            />
                            <p>
                                          Sort the restaurants in your area by process of elimination.
                            </p>
              </div>
              <div
                            className="icon-container"
              >
                            <img
                                          alt="Location"
                                          src="food-drink.png"
                            />
                            <p>
                                          Discover what you been craving.
                            </p>
              </div>
              <div
                            className="icon-container"
              >
                            <img
                                          alt="Location"
                                          src="smiley.png"
                            />
                            <p>
                                          Come back next time you're hungry!
                            </p>
              </div>
</section>,
            "className": "discover-info",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="icon-container"
>
                  <img
                                    alt="Location"
                                    src="hungry.png"
                  />
                  <p>
                                    Hungry but you can't put your finger on what you want?
                  </p>
</div>,
                <div
                  className="icon-container"
>
                  <img
                                    alt="Location"
                                    src="map.png"
                  />
                  <p>
                                    Sort the restaurants in your area by process of elimination.
                  </p>
</div>,
                <div
                  className="icon-container"
>
                  <img
                                    alt="Location"
                                    src="food-drink.png"
                  />
                  <p>
                                    Discover what you been craving.
                  </p>
</div>,
                <div
                  className="icon-container"
>
                  <img
                                    alt="Location"
                                    src="smiley.png"
                  />
                  <p>
                                    Come back next time you're hungry!
                  </p>
</div>,
              ],
              "className": "infographics d-flex",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt="Location"
                      src="hungry.png"
/>,
                    <p>
                      Hungry but you can't put your finger on what you want?
</p>,
                  ],
                  "className": "icon-container",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "Location",
                      "src": "hungry.png",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Hungry but you can't put your finger on what you want?",
                    },
                    "ref": null,
                    "rendered": "Hungry but you can't put your finger on what you want?",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt="Location"
                      src="map.png"
/>,
                    <p>
                      Sort the restaurants in your area by process of elimination.
</p>,
                  ],
                  "className": "icon-container",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "Location",
                      "src": "map.png",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Sort the restaurants in your area by process of elimination.",
                    },
                    "ref": null,
                    "rendered": "Sort the restaurants in your area by process of elimination.",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt="Location"
                      src="food-drink.png"
/>,
                    <p>
                      Discover what you been craving.
</p>,
                  ],
                  "className": "icon-container",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "Location",
                      "src": "food-drink.png",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Discover what you been craving.",
                    },
                    "ref": null,
                    "rendered": "Discover what you been craving.",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt="Location"
                      src="smiley.png"
/>,
                    <p>
                      Come back next time you're hungry!
</p>,
                  ],
                  "className": "icon-container",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "Location",
                      "src": "smiley.png",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Come back next time you're hungry!",
                    },
                    "ref": null,
                    "rendered": "Come back next time you're hungry!",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "section",
          },
          "type": "section",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
